### DISCUSSION TITLE
Add an editor similar to other popular programs using an .exe

### DISCUSSION THREAD

**Discussion by Loweredgames** [bevyengine/bevy#9110] [Q&A] [answered]
Hello I wanted to know if in the future an editor similar to Blender or other programs will be used. completely configurable but also usable for those who are not very savvy about code.
Thanks

**Comment by Loweredgames** [comment]
and even if an executable (.exe) like Godot is used

**Comment by NiklasEi** [comment, marked as answer]
The Bevy editor is still a much discussed feature (see #7100 ).
But I would say that Bevy strives to be usable for all people related to App/Game-development and as such also for not very code savvy people. As for the question for an executable: I would expect the editor to be distributed like that, yes.
Relevant part of the discussion linked above: https://github.com/bevyengine/bevy/discussions/7100#discussioncomment-4606657
